pages/osx/fdesetup.md

```diff
@@ -0,0 +1,24 @@
+# fdesetup
+
+> Set and retrieve FileVault related information.
+> Sudo required for most commands.
```

Comment:
```suggestion
```
Since **sudo** is required for all examples, I think we should omit this line

Reply:
Oh right. `sudo` is already written in all the examples.

Reply:
Sounds good. I will go ahead and remove that line.